lib/src/tasks/coverage/config.dart

```diff
@@ -17,6 +17,7 @@ library dart_dev.src.tasks.coverage.config;
 import 'package:dart_dev/src/tasks/config.dart';
 
 const bool defaultHtml = true;
+const List<String> defaultFunctionalTests = const [];
```

Comment:
This option should be a part of the test task configuration, not coverage. The coverage task uses the test configuration as well as its own